en/support/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Version Support
+menu: support
+lang: en
+---
+
+# Version Support
+
+Only the latest version of any given major release line is supported.
+
+Versions that are EOL (end of life) _may_ receive updates for critical security vulnerabilities, but the Express team offers no guarantee and has no intention of fixing any issues found or releasing any fixed versions.
+
+| Major Version | Support Start Date | Support End Date |
+| -- | -- | -- |
+| [**v5.x**{: style="color: gray; opacity: 0.8" }](/{{page.lang}}/5x/api.html) | **not yet released**{: style="color: gray; opacity: 0.8" } | **not yet released**{: style="color: gray; opacity: 0.8" } |
+| [**v4.x**{: style="color: green" }](/{{page.lang}}/4x/api.html) | April 2014 | **ongoing**{: style="color: green; opacity: 0.8" } |
+| [**v3.x**{: style="color: red" }](/{{page.lang}}/3x/api.html) | October 2012 | July 2015 |
+| [**v2.x**{: style="color: red" }](/{{page.lang}}/2x/) | March 2011 | July 2012 |
+| **v1.x**{: style="color: red" } | December 2010 | March 2011 |
+| **v0.14.x**{: style="color: red" } | December 2010 | December 2010 |
+
+## Commercial Support Options
+
+If you are unable to update to the latest version of Express, please contact one of our partners to receive security updates:
+
+ - [HeroDevs Never-Ending Support](http://www.herodevs.com/support/express-nes?utm_source=expressjs&utm_medium=link&utm_campaign=express_eol_page)
```

Comment:
Currently this link gives a 404 but I assume is the expected until this PR is merged in our side.

Reply:
Great!